consensus/consensus_leader_msg.go

```diff
@@ -23,7 +23,7 @@ func (consensus *Consensus) constructAnnounceMessage() []byte {
 	consensusMsg := message.GetConsensus()
 	consensus.populateMessageFields(consensusMsg)
 	// n byte of block header
-	consensusMsg.Payload = consensus.block // TODO: send only block header in the announce phase.
```

Comment:
Why is this todo removed?

Reply:
we need to send block anyway,  do it in announce stage or commit stage. If only send block headers, we cannot verify transactions